lib/beaker-http/http.rb

```diff
@@ -3,22 +3,26 @@ module Http
 
     # Beaker Http objects are essentially wrappers for a connection object that
     # utilizes a Beaker::Host object for easier setup during testing.
-    class Http
+    class Connection
       include Beaker::Http::Helpers
       extend Forwardable
 
-      attr_accessor :connection, :host
+      attr_accessor :connection
+      attr_reader :host
 
       def initialize(host)
```

Comment:
I'm about to open a PR for a utility function that may make an http request to a host that won't always be a beaker host. It might be a good thing to add a case for `is_a?(String)` that treats it as a hostname.